src/kernels/MIOpenLRNFwd.cl

```diff
@@ -410,10 +410,12 @@ MIOpenLRNAcrossChannels4(const __global _FLOAT* bottom,
     int b               = get_global_id(2); // batch
     MLO_READ_TYPE accum = 0;
     MLO_READ_TYPE bot_in2[MLO_LRN_KERNEL_SZ];
+    MLO_READ_TYPE bot_in[MLO_LRN_KERNEL_SZ];
     int c_i = 0, c_o = 0;
     for(int i = 0; i < MLO_LRN_KERNEL_SZ; ++i)
     {
         bot_in2[i] = 0;
+        bot_in[i] = 0;
```

Comment:
Hi @densamoilov, first of all, thank you very much for raising this PR!
We have some issues with Clang Format Tidy with this line and several lines below, likely due to a tab in front of "=" sign.
The best way to get rid of the format issue is to follow procedure outlined here:
https://github.com/ROCmSoftwarePlatform/MIOpen/wiki/How-to-format-code 

Thanks!

Reply:
Thanks for pointing this out! I fixed the format issues.